stable/minio/values.yaml

```diff
@@ -48,11 +48,18 @@ priorityClassName: ""
 existingSecret: ""
 accessKey: "AKIAIOSFODNN7EXAMPLE"
 secretKey: "wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY"
-configPath: "/root/.minio/"
-configPathmc: "/root/.mc/"
+configPath: "/etc/minio/"
```

Comment:
`--config-dir` has been deprecated from MinIO server as of minio/minio#7033, can you please remove this option completely?